--- reddit.js	2014-01-15 10:57:14.272155599 +0000
+++ /home/pi/reddit/reddit.js	2014-01-15 10:58:34.129996514 +0000
@@ -32,8 +32,6 @@
               if (resp && resp.data) {
                 callback(undefined, resp.data);
               }
-            } else {
-              console.log("pass something");
             }
           } else {
             if (!(typeof callback === "undefined") && typeof callback === "function") {
